cmds/exp/pox/pox.go

```diff
@@ -277,21 +295,46 @@ func poxRun(args []string) error {
 	return nil
 }
 
+func extraMounts() error {
+	if *extra == "" {
```

Comment:
minor, but i'd put this *extra handling down next to *creat and *run, for symmetry reasons.